lib/vanagon/project.rb

```diff
@@ -286,9 +297,48 @@ def get_preinstall_actions(pkg_state)
       end
     end
 
+    # Collects the install trigger scripts for the project for the specified packing state
+    #
+    # @param pkg_state String the package state we want to run the given scripts for.
+    #  Can be one or more of the 'install' or 'upgrade'
+    # @return {} of scriptlets to execute during the pkg_state (install or upgrade)
+    #   there can be more than one script for each package (key)
+    def get_install_trigger_scripts(pkg_state)
+      triggers = Hash.new { |hsh, key| hsh[key] = [] }
+      check_pkg_state(pkg_state, __method__)
+      pkgs = @components.map(&:install_triggers).flatten.compact.select { |s| s.pkg_state.include? pkg_state }
+      pkgs.each do |package|
+        triggers[package.pkg].push package.scripts
+      end
+      return triggers
```

Comment:
Explicit return is discouraged

Reply:
yeah, you should be able to replace this with just `triggers`